internal/interpreters/r_interpreter_functional_test.go

```diff
@@ -0,0 +1,179 @@
+package interpreters
+
+// Copyright (C) 2025 by Posit Software, PBC.
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/posit-dev/publisher/internal/logging"
+	"github.com/posit-dev/publisher/internal/types"
+	"github.com/posit-dev/publisher/internal/util"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/suite"
+)
+
+// RInterpreterFunctionalSuite contains functional tests that actually
+// interact with real R installations
+type RInterpreterFunctionalSuite struct {
+	suite.Suite
+	testProjectDir     util.AbsolutePath
+	log                logging.Logger
+	fs                 afero.Fs
+	interpreter        RInterpreter
+	rExecutable        util.AbsolutePath
+	defaultInterpreter *defaultRInterpreter
+	renvInstalled      bool
+}
+
+func TestRInterpreterFunctionalSuite(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+
+	suite.Run(t, new(RInterpreterFunctionalSuite))
+}
+
+func (s *RInterpreterFunctionalSuite) SetupTest() {
+	// Set up the test directory
+	parentDir, err := os.MkdirTemp("", "r-interpreter-test-*")
```

Comment:
I think you can use `afero` to create this temporary directory. I haven't used `afero` in a while, so I could be mistaken.

Reply:
_nods_ I looked into `afero` when I started and didn't find a way to do this easily (or as easily as ^^^). Mind if we punt on this refinement? I can add a TODO, but I think it's better to get this fix out than it is to be perfect here. Thoughts?